### PR TITLE
Show route

### PR DESCRIPTION
**Pull request by wittjosiah**

### CODE REVIEW

File: web/public/javascripts/init.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ let app = Elm.Main.embed(node);
 
 let mymap;
 let markers = {};
-let polylines = [];
+let polyline;
```
Comment (kiambogo) [inline web/public/javascripts/init.js]: Will this declare it as an empty list? or some null type?

Comment (kiambogo) [inline web/public/javascripts/init.js:6]: Will this declare it as an empty list? Or some null type?

File: web/src/Main.elm [hunk 2]
```diff
@@ -104,14 +101,22 @@ decodeSegment =
...
-      , page = MainPage
-      }
-    , up True
-    )
+    let
+        host =
+            if location.host == "localhost:9000" then
+                "http://localhost:9001"
+            else
+                "https://api.roadquality.org"
```
Comment (kiambogo) [inline web/src/Main.elm:109]: nice

File: web/public/javascripts/init.js (hunk 1 above)
Comment (wittjosiah) [reply web/public/javascripts/init.js:6]: null type, it doesn't actually need to be a list since the polyline returned includes the previous polyline

**Pull request merged**